Documentation/General/BuildSystem/HowToBuildLinux.md

```diff
@@ -0,0 +1,39 @@
+How to build for Linux {#HowToBuildLinux}
```

Comment:
Add a page to the Wiki:
https://github.com/ezEngine/ezEngine/wiki/Building-ezEngine